### ISSUE TITLE
Doesnt show crafting recipe

### ISSUE THREAD

**Issue by ilQueen** [philoufilou/Logic-Chips#4]
Hi,
I run Neoforge 1.20.6 server and I see that the items are in the list but it does not show the recipes for them.
I also use emi since emi and rei are the only ones available for 1.20.6

**Comment by ZZZank** [comment]
Confirmed, and there will be errors in the form of 
```
com.google.gson.JsonParseException: No key id in MapLike[{"item":"logicchips:chip","count":4}]
```
in latest.log on my side

**Comment by ZZZank** [comment]
closing this issue since it's fixed in 2.1.1
